politeiawww/api/cms/v1/v1.go

```diff
@@ -501,6 +501,7 @@ type UserDetailsReply struct {
 
 // EditUser edits a user's preferences.
 type EditUser struct {
+	UserID             string          `json:"userid"`           // The ID of the user being Edited
```

Comment:
So for a user editing their on details, they still have to include their own UserID?